File: perl/src/tests/t_prealloc_count.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$count = Atrshmlog::get_prealloc_buffer_count();

print "prealloc count : $count : \n";

$oldid = Atrshmlog::set_prealloc_buffer_count(42);

print "prealloc count : $oldid : \n";

$count = Atrshmlog::get_prealloc_buffer_count();

print "prealloc count : $count : \n";

$oldid = Atrshmlog::set_prealloc_buffer_count(128);

print "prealloc count : $oldid : \n";

$count = Atrshmlog::get_prealloc_buffer_count();

print "prealloc count : $count : \n";


print " \n";

exit (0);

# end of main

File: python/src/tests/t_prealloc_count.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

count = atrshmlog.get_prealloc_buffer_count()

print('prealloc count : ' + str(count) + ' : ')

oldid = atrshmlog.set_prealloc_buffer_count(42)

print('prealloc count : ' + str(oldid) + ' : ')

count = atrshmlog.get_prealloc_buffer_count()

print('prealloc count : ' + str(count) + ' : ')

oldid = atrshmlog.set_prealloc_buffer_count(128)

print('prealloc count : ' + str(oldid) + ' : ')

count = atrshmlog.get_prealloc_buffer_count()

print('prealloc count : ' + str(count) + ' : ')


print (' ')

exit(0)

# end of test

